package.json

```diff
@@ -9,8 +9,8 @@
         "bootstrap": "yarn lerna bootstrap",
         "clean": "yarn lerna clean --yes",
         "build": "yarn lerna run build",
-        "lint": "./node_modules/eslint/bin/eslint.js packages/*/src --ext .ts --fix ",
-        "lint:tests": "./node_modules/eslint/bin/eslint.js __tests__ packages/*/__tests__ --ext .ts --fix",
+        "lint": "eslint packages/*/src --ext .ts --fix ",
+        "lint:tests": "eslint __tests__ packages/*/__tests__ --ext .ts --fix",
```

Comment:
Can you remove `--fix` flag and add `--max-warnings 0`? Then [CI will fail] and there will be no way to merge code with lint errors.

[CI will fail]: https://github.com/ArkEcosystem/core/blob/develop/.github/workflows/lint.yml#L37-L38